ui/splash/src/commonMain/kotlin/social/androiddev/splash/composables/SplashScreen.kt

```diff
@@ -0,0 +1,48 @@
+package social.androiddev.splash.composables
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import social.androiddev.splash.navigation.SplashComponent
+
+@Composable
+fun SplashScreen(
+    modifier: Modifier,
+    component: SplashComponent
+) {
+    SplashScreen(
+        modifier = modifier,
+        navigateToWelcome = {
+            component.isUserLoggedIn(false)
```

Comment:
is it intended to have this prediction checked inside the ui? I thought decompose will decouple navigation logic from the ui.
Maybe other targets have different navigation behaviour

Reply:
This is outdated dummy code. It has been replaced in next commit